gh-pages/index.md

```diff
@@ -31,6 +31,7 @@ layout: index
 * [Scroll Options](plugins/scroll-options/test/index.html)
 * [Serialize Disabled Interactions](plugins/serialize-disabled-interactions/test/index.html)
 * [Shadow Block Converter](plugins/shadow-block-converter/test/index.html)
+* [Suggested Blocks](plugins/suggested-block/test/index.html)
```

Comment:
There is a typo here, link is incorrect. Should be suggested-blocks with a 's' at the end